pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1317,7 +1317,7 @@ protected void handleProducer(final CommandProducer cmdProducer) {
                                 }
                                 createInitSubFuture =
                                         topic.createSubscription(initialSubscriptionName, InitialPosition.Earliest,
-                                                false);
+                                                false, null);
```

Comment:
Could we add a new method or use the empty map to avoid putting the `NULL` value directly.

Reply:
I think that adding a new method will make it more messy than it already looks like.